### ISSUE TITLE
Can't export .dds files

### ISSUE THREAD

**Issue by VLPES** [Ryder25/Ego-Engine-Modding#25]
Hello,
I'm trying to export a helmet file from 'incardriver.erp' (using F1 2018 btw) but I can't seem to get it to work. Likely because I don't actually know what I'm supposed to be doing. What I've done is placed the corresponding mipmaps file in a folder on my desktop, tried to extract the helmet file to this folder, and used the mipmaps in that folder as the one to save it with. I keep getting an error, however, which states that I couldn't export it, either because the mipmaps is modded wrong or it's in the wrong place.

Basically, I don't know what I'm doing and some help would be appreciated.

Cheers.

**Comment by VLPES** [comment]
Nevermind.
